test/relayclient/RelayClient.test.ts

```diff
@@ -179,25 +178,25 @@ contract('RelayClient', function (accounts) {
 
   describe('#relayTransaction()', function () {
     it('should warn if called relayTransaction without calling init first', async function () {
-      relayClient = new RelayClient(underlyingProvider, gsnConfig)
-      sinon.spy(relayClient, '_warn')
+      const relayClient1 = new RelayClient(underlyingProvider, gsnConfig)
```

Comment:
Why?

Reply:
I mean, the only change I can see here is rename `relayClient` to `relayClient1`.

Reply:
Right. I'll remove it (it was changed a lot of times, and finally turned to it's original structure...)

Reply:
Will be refu removed (the code was changed several times, and eventually ended up where it started...)